publisher.rb

```diff
@@ -0,0 +1,38 @@
+require 'aws-sdk-sqs'
```

Comment:
Keep standard indentation.
Better still use Ruby Style Guide - https://github.com/rubocop-hq/ruby-style-guide

Reply:
Standardize and keep spaces only. There is a mix of tabs and spaces..